frontend/frontend.c

```diff
@@ -656,15 +697,12 @@ int main(int argc, char **argv)
     else
         audioReadSize = AUDIO_BUF_SIZE;
 
-    // Calculate the size and number of frames we are going to encode
-    if (sfinfo.frames)
-        total_frames = (sfinfo.frames -1) / TWOLAME_SAMPLES_PER_FRAME +1;
-
 
     // Now do the reading/encoding/writing
     while ((samples_read = inputfile->read(inputfile, pcmaudio, audioReadSize)) > 0) {
         int bytes_out = 0;
 
+#if 0
```

Comment:
Can this be deleted instead of the code being disabled?

Reply:
Oh, of course yes, it can! I forgot to delete that bunch of statements.